### DISCUSSION TITLE
GameUserSettings.ini overwritten on restart

### DISCUSSION THREAD

**Discussion by abighairyspider** [wolveix/satisfactory-server#118] [Q&A] [answered]
I'm trying to disable the seasonal events in GameUserSettings.ini, but every time I restart this file is getting overwritten. Is there any way to make this file persist? I've tried copying to the root of /config, but it looks like it's being ignored.

**Comment by wolveix** [comment, marked as answer]
Please use issues for this in the future :)

There's an environment variable to disable seasonal events: 

```
DISABLESEASONALEVENTS=true
```

**Reply by msladek** [reply]
@wolveix We can still turn this into a discussion ;D

should we mention in the README that console commands will be overwritten on container restart?

**Reply by wolveix** [reply]
Yeah, we probably should. You used to be able to manually provide your own .ini files, but we disabled that for a reason that currently escapes me.
